### PR TITLE
TSK-778 - Check if exist the history_event's table before running the script

### PR DESCRIPTION
**Pull request by ignaciorecuerda**

### CODE REVIEW

File: rest/taskana-rest-spring-example/src/main/resources/sql/sample-data/check-history-event-exist.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SET SCHEMA TASKANA;
```
Comment (MiguelMartinRojas) [inline rest/taskana-rest-spring-example/src/main/resources/sql/sample-data/check-history-event-exist.sql]: TASKANA should be variable.

File: rest/taskana-rest-spring-example/src/main/java/pro/taskana/sampledata/SampleDataGenerator.java [hunk 2]
```diff
@@ -110,12 +113,26 @@ public void generateSampleData(String schemaName) {
         }
 
         runner.setStopOnError(true);
+
+        String[] script = {WORKBASKET, DISTRIBUTION_TARGETS, CLASSIFICATION, TASK, ATTACHMENT, WORKBASKET_ACCESS_LIST,
```
Comment (MiguelMartinRojas) [inline rest/taskana-rest-spring-example/src/main/java/pro/taskana/sampledata/SampleDataGenerator.java]: Move this logic into a function like getScriptsList

File: rest/taskana-rest-spring-example/src/main/resources/sql/sample-data/check-history-event-exist.sql (hunk 1 above)
Comment (ignaciorecuerda) [reply rest/taskana-rest-spring-example/src/main/resources/sql/sample-data/check-history-event-exist.sql]: In this case, this property is unnecessary.
Removed!

Comment (MiguelMartinRojas) [reply rest/taskana-rest-spring-example/src/main/resources/sql/sample-data/check-history-event-exist.sql]: no, it should be variable, not removed. find schemaName into the taskana-core

**Pull request merged**